third_party/asr/asr595x/asr_riscv.gni

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("asr_board.gni")
+
+riscv_arch = "rv32imafc"
+riscv_abi = "ilp32f"
+#  riscv_cpu = ""
```

Comment:
delete commented out lines or add some comment on why these should exist commented out